.github/ISSUE_TEMPLATE/bug.yml

```
@@ -22,6 +23,12 @@ body:
 
         crate.yml contents:
 
+  - type: textarea
+    id: problem_description
+    attributes:
+      label: Problem description
+      description: Succinct description of the problem
```

Maybe:
```suggestion
      description: Concise description of the problem
```
? 
as I didn't know `succinct` myself :) :)